spec/factories/messages.rb

```diff
@@ -0,0 +1,7 @@
+FactoryBot.define do
+  factory :message do
+    association :user
+    association :conversation
```

Comment:
You can just call 
```
factory :message do
  user
  conversation
   ...
```
Instead of calling `association :youFactory`, is an alias: https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#aliases